### PR TITLE
Create an extractDiff helper to create a FilterQuery

### PR DESCRIPTION
**Pull request by elizabethengelman**

### CODE REVIEW

**Comment by paytonrules** [comment]
I don't love the name "helpers" for the package. Given that it only has one public function, maybe this could just live in cmd/helpers.go as part of the cmd package?Or maybe something to do with filter queries.

**Comment by elizabethengelman** [comment]
@paytonrules Yeah, that is fair - I was wondering if there were other "helper" type methods we could extract from other commands? 

Also, I had initially put it in the streamer package, but that felt weird as well, since it was so tied to viper.

**Review by paytonrules — APPROVED** [review]
Other than the package name this looks great.

**Comment by paytonrules** [comment]
After this PR it might be good to create a story to run extract-diffs in CI as well. It wouldn't have caught the original bug (because the empty contracts limped along) but it would catch that mistake now.

File: cmd/helpers/helpers.go [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+	logrus.Infof("Watched addresses: %s", addressesToLog)
+}
+
+func getAddressesFromViper() ([]string, error) {
+	contracts := viper.GetStringMapString("contract")
+	var addressStrings []string
+	for contractName := range contracts {
+		address, ok := viper.GetStringMapString("contract." + contractName)["address"]
+		if !ok {
+			parseErr := errors.New(fmt.Sprintf("%s not parsed properly into viper", contractName))
```
Comment (rmulhol) [inline cmd/helpers/helpers.go]: super minor but could use `fmt.Errorf` here

File: cmd/helpers/helpers_test.go [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+		}
+
+		It("creates a filter query with contract addresses from viper config", func() {
+			viper.SetConfigType("toml")
+			readConfigErr := viper.ReadConfig(bytes.NewBuffer(testConfig))
+			Expect(readConfigErr).NotTo(HaveOccurred())
+
+			filterQuery, filterQueryErr := helpers.CreateFilterQuery()
+			Expect(filterQueryErr).NotTo(HaveOccurred())
+			Expect(filterQuery.Addresses).To(ContainElements(contractAddresses))
```
Comment (rmulhol) [inline cmd/helpers/helpers_test.go]: maybe `ConsistOf` instead of `ContainElements`?

**Review by rmulhol — APPROVED** [review]

**Pull request merged**